app/src/org/commcare/tasks/ProcessAndSendTask.java

```diff
@@ -326,6 +326,9 @@ private void sendForms(FormRecord[] records) throws TaskCancelledException {
                                 // A processing failure indicates that there there is no point in
                                 // trying that submission again immediately
                                 break;
+                            } else if (results[i] == FormUploadResult.RATE_LIMITED) {

```

Comment:
@ctsims Where are we setting the result to `RATE_LIMITED` ?

Reply:
whoops, missed that delta in the commit, thanks. Added the callout here